client/components/RoomIcon/OmnichannelRoomIcon/OmnichannelCoreSourceRoomIcon.tsx

```diff
@@ -27,5 +27,5 @@ export const OmnichannelCoreSourceRoomIcon = ({
 	size: ComponentProps<typeof Icon>['size'];
 }): ReactElement => {
 	const icon = iconMap[room.source?.type || 'other'] || 'headset';
-	return <Icon name={icon} size={size} color={colors[room.v.status || 'offline']} />;
+	return <Icon name={icon} size={size} color={colors[room.v?.status || 'offline']} />;
```

Comment:
Again, unrelated change, just for safety in the case a room doesn't have a `v` prop. This doesn't happen very often, but if it happens the UI will be unusable.